sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -4139,8 +4139,8 @@ def test_pretty_Union_issue_10414():
 def test_pretty_Intersection_issue_10414():
     x, y, z, w = symbols('x, y, z, w')
     a, b = Interval(x, y), Interval(z, w)
-    ucode_str = '[x, y] ∩ [z, w]'
-    ascii_str = '[x, y] n [z, w]'
+    ucode_str = '[Max(x, z), Min(w, y)]'
+    ascii_str = '[Max(x, z), Min(w, y)]'
```

Comment:
This is supposed to be a test of printing an Intersection so if it doesn't give an intersection then something other than interval should be used as the arguments